exampleSite/config.yaml

```diff
@@ -111,7 +117,8 @@ params:
     - icon: ti-twitter
       name: "@UNICEFinnovate on Twitter"
       link: https://twitter.com/UNICEFinnovate
-
+# github base link for edit page feature
+  editPrefix: https://github.com/unicef/inventory-hugo-theme/blob/main/exampleSite/content
```

Comment:
`editPrefix` appears three times in the config file. Please reduce it to a single invocation. I suggest keeping it towards the top of the `params`, separate from the `brand` section.

Reply:
I have this fixed this and added the editPrefix at the top of `params `  section.

Reply:
> 1. **HTML 5 validation**: There are still HTML validation errors. It appears one of the colors is not being loaded correctly. Could you look into this more? _Hint_: The [W3 Validator](https://validator.w3.org/) is a useful tool. You can paste HTML from your browser's Developer Tools for quick validation while the Hugo server is running.

I tried pasting the html source code of a page and it showed something different from the above. I am unable to figure out the exact place where this errors occured as i dont know the code you pasted in W3 validator. The below three includes a anchor tag containing UNICEF, this change is not something i did. Could please provide the code you pasted, so that i can figure where it's going wrong.

> 2. **Reduce commit footprint**: There are additional, unrelated changes committed to this Pull Request. We want to keep the PR scope lean and focused, and other minor changes should not be committed. This speeds up the review process for future changes too.
This PR only includes changes which are relevant.
The commit history include previous commits. I exactly don't know why this happened, may be this is because i did some changes in main and they got into this branch while doing merge.
I tried doing a git rebase and squashing all the commits but i got some conflits. so i aborted it.

Reply:
The HTML validation issues were an issue with my environment. I tried again and the errors cleared. Not sure what happened the first time.

I did a manual rebase now and squashed the commits down to a single commit. This one was especially tricky. It will be worthwhile to review git rebasing together in terms of workflows. But I think the circumstances were unusual here since the change was pending for some time, and other changes entered the main code base.